text/0000-eliminate-finalize.md

```diff
@@ -0,0 +1,142 @@
+
+- Title: Eliminate Finalize Step
+- Authors: [David Burkett](mailto:davidburkett38@gmail.com)
+- Start date: Aug 09, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000) 
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+Use one-time addresses which include a pre-generated public excess and public nonce in order to eliminate the 3rd step in transaction building (finalize) [1].
+
+# Motivation
+[motivation]: #motivation
+
+To simplify transaction building.
```

Comment:
Would be good to understand _how_ transaction building gets simplified with this RFC. What are the specific use cases that become easier? Does it become easier for services to process payments? Will user <> user flows be simplified? Cold storage transfers?

I also understand it as enabling payment proofs in the invoice flow, something which hasn't been possible before and is [an open question of RFC006](https://github.com/mimblewimble/grin-rfcs/blob/master/text/0006-payment-proofs.md#unresolved-questions), which might be worth while mentioning here as well.

Reply:
Yea, was focusing on the technical details. I got a bit lazy. (ok, a lot lazy)

Reply:
Haha, understandably! I can help here, if we can identify roughly what these are. It's not entirely clear to me where the benefits arise